source/_includes/installation/container.md

```diff
@@ -52,7 +52,7 @@ If you change the configuration, you have to restart the server. To do that you
   content: |
 
     ```bash
-    docker compose restart
+    docker-compose restart
```

Comment:
This change is not correct. If you still need to use this, it means you are using a very old version of Docker.

../Frenck